init.lua

```diff
@@ -263,10 +263,6 @@ local function anvil_rotate(pos, node, user, mode, new_param2)
 		local inv = player:get_inventory()
 		local wielded = inv:get_stack(wield_list, wield_index)
 
-		if wielded:get_name() ~= "screwdriver:screwdriver" then
```

Comment:
Use a local variable after L255 to contain the wielded item name and check whether the player is still holding it here. That's relevant to have a sane `anvil_rightclick` call below.

Reply:
Fixed. Sorry for the delay, I was on vacation.